packages/app/src/app/pages/Live/BlinkingDot.tsx

```diff
@@ -12,7 +12,8 @@ const DotContainer = styled.div`
   }
 `;
 
-export class BlinkingDot extends React.PureComponent {
+export class BlinkingDot extends React.PureComponent<{}, { showing: boolean }> {
+  timer: NodeJS.Timer;
```

Comment:
I believe in the browser case we need `number` here. Then use `this.timer = window.setInterval`.

Reply:
Not blocking from merging, but for reference later.

Reply:
Ah, yeah, hm... weird typing behaviour. Was not aware that `window.setInterval` and `setInterval` was typed differently. Fixed